### PR TITLE
Refine test payloads

### PR DESCRIPTION
**Pull request by jkoenig134**
# Readiness checklist

-   [ ] I added/updated tests.
-   [x] I ensured that the PR title is good enough for the changelog.
-   [x] I labeled the PR.

### CODE REVIEW

File: packages/runtime/test/lib/testUtils.ts [hunk 1]
```diff
@@ -162,9 +162,7 @@ export async function syncUntilHasEvent<TEvent extends Event>(
 
 export async function uploadOwnToken(transportServices: TransportServices, forIdentity?: string): Promise<TokenDTO> {
     const response = await transportServices.tokens.createOwnToken({
-        content: {
-            content: "Hello"
-        },
+        content: { aKey: "aValue" },
```
Comment (britsta) [inline packages/runtime/test/lib/testUtils.ts:165]: Mostly, we start our example values with capital letters, i.e. `"AValue"`.

File: packages/runtime/test/transport/tokens.test.ts [hunk 2]
```diff
@@ -39,9 +39,7 @@ describe("Tokens", () => {
 describe("Tokens errors", () => {
     test("create a token with 'expiresAt' set to undefined", async () => {
         const response = await runtimeServices1.transport.tokens.createOwnToken({
-            content: {
-                content: "Hello"
-            },
+            content: { aKey: "aValue" },
```
Comment (britsta) [inline packages/runtime/test/transport/tokens.test.ts:42]: Mostly, we start our example values with capital letters, i.e. `"AValue"`.

**Review by britsta — CHANGES_REQUESTED** [review]
Could you please also replace the "Döner" test description in the `IQLQuery.test.ts`? 😂 I've been wanting to do this for a while...

**Comment by jkoenig134** [comment]
> Could you please also replace the "Döner" test description in the `IQLQuery.test.ts`? 😂 I've been wanting to do this for a while...

Döner??????

File: packages/runtime/test/lib/testUtils.ts (hunk 1 above)
Comment (jkoenig134) [reply packages/runtime/test/lib/testUtils.ts:165]: actually we don't :D

Comment (britsta) [reply packages/runtime/test/lib/testUtils.ts:165]: Oops, then we'll probably do it this way or that currently 😂 Therefore, never mind for now 🌝

Comment (jkoenig134) [reply packages/runtime/test/lib/testUtils.ts:165]: @britsta better?

**Comment by britsta** [comment]
> Döner??????

A test title that hunts me in my sleep 🌮

**Comment by jkoenig134** [comment]
> > Döner??????
> 
> A test title that hunts me in my sleep 🌮

actually Döner was just a value to test that was correctly placed in `'`s, but Döner is a bad naming for this test input 👍🏼

File: packages/runtime/test/lib/testUtils.ts (hunk 1 above)
Comment (britsta) [reply packages/runtime/test/lib/testUtils.ts:165]: Certainly better :D Even if there are still values that start with capital letters. But I don't think we need to do it all in this PR, the main thing is that the hello's and the kebab have disappeared 😂

**Review by britsta — APPROVED** [review]

File: packages/runtime/test/lib/testUtils.ts (hunk 1 above)
Comment (jkoenig134) [reply packages/runtime/test/lib/testUtils.ts:165]: starting with capital a?

**Pull request merged**